src/openrct2-cli/Cli.cpp

```diff
@@ -20,7 +20,7 @@ using namespace OpenRCT2;
 int main(int argc, const char** argv)
 {
     int32_t rc = EXIT_SUCCESS;
-    int runGame = cmdline_run(argv, argc);
+    int runGame = CmdlineRun(argv, argc);
```

Comment:
Maybe `CmdLineRun` (uppercase L) is better, since they are separate words. The same goes for all other cases of `Cmdline...`. This can be left for when this gets refactored to namespaces later on.

Reply:
Could you create an issue for it?

Reply:
Done: https://github.com/OpenRCT2/OpenRCT2/issues/19273